src/components/container/Header.tsx

```diff
@@ -1,33 +1,41 @@
-import * as React from 'react'
+import React, {ChangeEvent} from 'react'
 import TodoList from "./todoList";
 import {useState} from "react";
-import {useCustom} from "../hooks/customHooks";
+import {addTodo} from '../redux/todoSlice';
+import {useAppDispatch} from "../redux/store";
 
-export default function Header () {
+export default function TodoListContainer() {
 
-    const [todoInput, setTodoInput] = useState('')
-    const [todos, setTodos] = useCustom("todo" )
-    const [durings, setDurings] =useCustom("during")
-    const [dones, setDones] =useCustom("done" )
+    const [value, setValue] = useState('');
+    const dispatch = useAppDispatch();
 
-    const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-      setTodoInput(event.target.value)
-    }
-    const submitTodo = () => {
-      setTodos([
-          ...todos,
-          {text: todoInput, id: Math.random() * 1000}
-      ])
-    setTodoInput('')
-    }
+    const onSubmit = (event:ChangeEvent<HTMLFormElement>) => {
+        event.preventDefault();

```

Comment:
Komentarz rozwiązany a dalej jest nie spójnie linia  z `setValue('')` też nie ma `;`.
zasada od tego z eslinta to: https://eslint.org/docs/latest/rules/semi